lib/go/thrift/socket_aix_syscall.go

```diff
@@ -0,0 +1,29 @@
+//go:build aix
+// +build aix
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership. The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package thrift
+
+import "syscall"
+
+func recvfromNonblocking(fd int, p []byte) (int, syscall.Sockaddr, error) {
```

Comment:
looks good to me, but I have some nitpick on the  naming here.

this function is not just recvfrom nonblocking, we also always apply the peek flag, so a more accurate name is probably `peekNonblocking`. what do you think?

Reply:
Yes, good point, I think you're right. I've renamed it to `peekNonblocking()`.